scripts/update-links.js

```diff
@@ -0,0 +1,48 @@
+import { S3Client, ListObjectsV2Command } from '@aws-sdk/client-s3';
+import { Linker } from 'nodejs-latest-linker/common.js';
+import { writeFile } from 'node:fs/promises';
+
+const ENDPOINT =
+  'https://07be8d2fbc940503ca1be344714cb0d1.r2.cloudflarestorage.com';
+const BUCKET = 'dist-prod';
+const RELEASE_DIR = 'nodejs/release/';
+const DOCS_DIR = 'nodejs/docs/';
+
+const client = new S3Client({
+  endpoint: ENDPOINT,
+  region: 'auto',
+  credentials: {
+    accessKeyId: process.env.CF_ACCESS_KEY_ID,
+    secretAccessKey: process.env.CF_SECRET_ACCESS_KEY,
+  },
+});
+
+(async function main() {
+  const allDirs = await listDirectory(RELEASE_DIR);
+  const linker = new Linker({ baseDir: RELEASE_DIR, docsDir: DOCS_DIR });
+  const links = await linker.getLinks(allDirs, dir => listDirectory(`${dir}/`));
+  await writeFile(
+    './src/constants/links.json',
```

Comment:
Nit but could you make the file name a little bit more specific like `redirects.json` or `redirectLinks.json`? Same with the Github action and script name just so it's a little bit more clear